docs/csharp/language-reference/operators/assignment-operator.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "= operator - C# reference"
+title: "Assignment operators - C# reference"
```

Comment:
```suggestion
title: "Assignment operator - C# reference"
```

Reply:
Plural because of the compound operators